<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRCODEAPP</class>
 <widget class="QMainWindow" name="QRCODEAPP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Lookup">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Lookup QR Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="APP_Quit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
   </widget>
   <widget class="QLabel" name="QRDisplayBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>184</width>
      <height>184</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>QRCODE/image.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
